lua/swenv/api.lua

```diff
@@ -117,6 +117,19 @@ local get_conda_base_path = function()
   end
 end
 
+local get_conda_base_env = function ()
+  local venvs = {}
+  local path = os.getenv("CONDA_EXE")
+  if path then
```

Comment:
nice, seems better to use `os.getenv` since it returns `nil` compared to the `vim.fn.getenv` above :+1: